_posts/2019-09-05-cover-song-similarity.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: post
+title: Cover song similarity algorithms in Essentia
+category: news
+---
+
+
+`Cover Song Identification` or `Music Version Identification` is a task of identifying when two musical recordings are derived from the same music composition. A cover version of a song can be drastically different from the original recording. It may have variations in key, tempo, structure, melody, harmony, timbre, language and lyrics compared to its original version, which makes it a challenging task [1]. It is typically set as a query-retrieval task where the system retrieves a ranked list of possible covers of a given query song from a reference database. We recommend you reading [1] for more comprehensive details of this task. 
+
+Most of the state-of-the-art cover identification systems use pairwise comparison of two musical recordings to compute a similarity distance by using the following workflow.
```

Comment:
"use pairwise comparison of music recordings computing a similarity distance with the following workflow:" ?

Reply:
@dbogdanov What about this?

```Most of the state-of-the-art cover identification systems undergoes the folowing steps to compute a similarity distance using pairwise comparison of given music recordings. ie, A query song is compared against each and every song in the music database to retrieve a ranked-list of music where an element in the first index is most simialr song to the query song.```